<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContinuousSnapshotWindowSelectionPage</class>
 <widget class="QWizardPage" name="ContinuousSnapshotWindowSelectionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Window Selection</string>
  </property>
  <property name="subTitle">
   <string>Please select target windows and specify how you want to output files.</string>
  </property>
  <attribute name="pageId">
   <string notr="true"/>
  </attribute>
  <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,1,0">
   <item>
    <widget class="QLabel" name="targetLabel">
     <property name="text">
      <string>Target windows</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="targetListWidget">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="outputGroupBox">
         <property name="title">
          <string>Output file</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="onefileRadioButton">
            <property name="text">
             <string>In one file</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="respectivelyRadioButton">
            <property name="text">
             <string>Respectively</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="layoutGroupBox">
         <property name="title">
          <string>Layout</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="asisRadioButton">
            <property name="text">
             <string>As is</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="horizontallyRadioButton">
            <property name="text">
             <string>Horizontally</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="verticallyRadioButton">
            <property name="text">
             <string>Vertically</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="backgroundGroupBox">
         <property name="title">
          <string>Background</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QRadioButton" name="whiteRadioButton">
            <property name="text">
             <string>white</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="transparentRadioButton">
            <property name="text">
             <string>transparent</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>respectivelyRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>layoutGroupBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
